sqlengine/mysql_engine.go

```diff
@@ -34,6 +34,11 @@ func (d *MySQLEngine) Open(address string, port int64, dbname string, username s
 	return nil
 }
 
+func (d *MySQLEngine) Ping() error {
+	panic("Not implemented")
```

Comment:
Why can't we do a `d.db.Ping()` here as well? It's a function defined by Go's SQL package, not by a specific driver.